src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -370,6 +408,33 @@ export class SavedObjectsService
           factory,
         });
       },
+      addEncryptionExtension: (factory) => {
```

Comment:
My understanding is that the wrapper system is still in tact. Even though we don't implement encryption, spaces, or security features via wrappers anymore, we can still use potentially create a wrapper for something else in the future. I don't think there are currently any other wrappers though.

Alternatively, we can get rid of the wrapper system altogether if we don't think there will be a need for it later.

Reply:
Got it, thanks, I'll defer that to the Core team then (I'd honestly remove as much stuff as possible if it's not used by us and we're not aware of any external plugins that rely on this).

Reply:
@elastic/kibana-core Should we remove the wrapper system? Maybe wait until the next major release if this would be a breaking change due to parameter changes?

Reply:
I think we would be fine removing is right now. It could only potentially breaks 3rd party plugins registering their own wrappers, and tbh the 'plugin API' is still experimental so we would be fine with it.

So +1 to 'remove as much stuff as possible'

Reply:
I'd say that getting to a point of clean code is higher priority here than trying to preserve compatibility for 3rd party plugins for an experimental (plugin API), widely unused (AFAIK no 3rd party plugins are leveraging the SO wrappers) API. 

So I would personally just remove anything that is no longer internally used. 

But please @rudolf tell me if you think I'm wrong.

Reply:
++ removing complexity is a bigger priority

Reply:
@legrego Any opposition to me getting back in there and stripping out the wrapper system entirely?